cmd/cape/cmd/delete.go

```diff
@@ -71,7 +80,13 @@ func doDelete(url string, functionID string, insecure bool, auth entities.Functi
 		if res.StatusCode == 404 {
 			return fmt.Errorf("function ID not found: %s", functionID)
 		}
-		return fmt.Errorf("expected 200, got server response code %d", res.StatusCode)
+
+		errMsg := ErrorMsg{}
+		if err := json.NewDecoder(res.Body).Decode(&errMsg); err != nil {
+			return fmt.Errorf("expected 200, got server response code %d and could not decode err msg", res.StatusCode)
+		}
+
+		return fmt.Errorf("expected 200, got server response code %d with msg %s", res.StatusCode, errMsg.Error)
```

Comment:
If we received a properly encoded error from the server, we should probably bubble that up directly.